src/components/Button/index.js

```diff
@@ -0,0 +1,69 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { cx } from "emotion"
+import {
+  wrapper,
+  primaryBtn,
+  secondaryBtn,
+  dangerBtn,
+  ghostBtn
+} from "./style.js"
+
+const ButtonPropTypes = {
+  className: PropTypes.string,
+  onClick: PropTypes.func,
+  disable: PropTypes.bool,
+  type: PropTypes.oneOf(["primary", "secondary", "danger"]),
+  fullWidth: PropTypes.bool,
+  size: PropTypes.oneOf(["small", "large"]),
+  ghost: PropTypes.bool,
+  href: PropTypes.string
```

Comment:
We won't be needing both href and onClick handler. They serve the same purpose. So let's use onClick and remove the href property.